cmd/main.go

```diff
@@ -19,7 +19,7 @@ import (
 )
 
 func main() {
-	conn, ch, err := amqp.NewConnection("amqp://guest:guest@localhost:5672")
+	conn, ch, err := amqp.NewConnection(os.Getenv("AMQP_DSN"))
```

Comment:
we could check if it is empty and default to `"amqp://guest:guest@localhost:5672"`